mmaction/models/localizers/ssn.py

```diff
@@ -0,0 +1,122 @@
+import torch
+import torch.nn as nn
+
+from .. import builder
+from ..registry import LOCALIZERS
+from .base import BaseLocalizer
+
+
+@LOCALIZERS.register_module
+class SSN(BaseLocalizer):
+
+    def __init__(self,
+                 backbone,
+                 cls_head,
+                 in_channels=3,
+                 spatial_type='avg',
+                 dropout_ratio=0.5,
+                 loss_cls=dict(type='SSNLoss'),
+                 train_cfg=None,
+                 test_cfg=None):
+
+        super(SSN, self).__init__()
+        self.backbone = builder.build_backbone(backbone)
+        self.cls_head = builder.build_head(cls_head)
```

Comment:
`self.backbone`, `self.cls_head`, `self.train_cfg = train_cfg`, `self.test_cfg`, `self.init_weights()` can be build in `localizers/base.py`. To change this, you may need to do some minor modification in `bmn` and `bsn` to avoid conflict.